app/views/forms/show.html.erb

```diff
@@ -15,9 +15,12 @@
       <p><%= flash[:message] %></p>
     <% end %>
 
-    <p class="govuk-!-margin-bottom-9">
-      <%= render PreviewLinkComponent::View.new(@form.pages, link_to_runner(Settings.forms_runner.url, @form.id, @form.form_slug)) %>
-    </p>
+    <% preview_link = PreviewLinkComponent::View.new(@form.pages, link_to_runner(Settings.forms_runner.url, @form.id, @form.form_slug)) %>
+    <% if preview_link.render? %>
+      <p class="govuk-!-margin-bottom-9">
+        <%= render preview_link %>
+      </p>
+    <% end %>
```

Comment:
Could we not wrap this up into the component itself? 

Ideally we want to call `<%= render PreviewLinkComponent::View.new...`  and let the component decide how its rendered or if its rendered

Reply:
That would be cool... is there a way to have it be an optional part of the component? We use it differently in different places:

```
$ git grep -C 1 'render PreviewLinkComponent'
app/views/forms/show.html.erb-    <p class="govuk-!-margin-bottom-9">
app/views/forms/show.html.erb:      <%= render PreviewLinkComponent::View.new(@form.pages, link_to_runner(Settings.forms_runner.url, @form.id, @form.form_slug)) %>
app/views/forms/show.html.erb-    </p>
--
app/views/live/show_form.html.erb-    <p>
app/views/live/show_form.html.erb:      <%= render PreviewLinkComponent::View.new(form.pages, link_to_runner(Settings.forms_runner.url, form.id, form.form_slug, mode: :preview_live)) %>
app/views/live/show_form.html.erb-    </p>
--
app/views/pages/index.html.erb-      <% end %>
app/views/pages/index.html.erb:      <%= render PreviewLinkComponent::View.new(@pages, link_to_runner(Settings.forms_runner.url, @form.id, @form.form_slug)) %>
app/views/pages/index.html.erb-    </div>

```

Reply:
🤔  just off the top of my head, could pass in an optional argument (default to include `<p>`) and then either include it or not in `view.rb`, Interesting to see there's a difference between `show.html.erb` and `show_live.html.erb`. Any chance we could standardise it?

Reply:
I think given the three places where we use this component we have it in different containers, it's more pain to move it in than not... are you happy with leaving it as is @aliuk2012?

Reply:
Sorry haven't had much time to spike something out to show you but happy to leave it as is and we can iterate at another time. t